src/helpers.js

```diff
@@ -1,6 +1,6 @@
 const p = require('path')
-const babel = require('babel-core')
-const template = require('babel-template')
+const babel = require('@babel/core')
+const template = require('@babel/template').default
```

Comment:
Template is already available at `babel.template` so you should be able to avoid this dependency at least.